Polygon_mesh_processing/doc/Polygon_mesh_processing/Polygon_mesh_processing.txt

```diff
@@ -511,6 +511,37 @@ input triangle mesh.
 \subsubsection InsideExample Inside Test Example
 \cgalExample{Polygon_mesh_processing/point_inside_example.cpp}
 
+
+\subsection PMPEnvelope Polyhedral Envelope Containment Check
+
+The class `CGAL::Envelope` provides functors that test whether a query point, segment or triangle
+is fully contained in a <em>polyhedral envelope</em> of a triangle mesh.
+
+\cgalFigureBegin{envelopeFig, envelope.png}
+Two polyhedral envelopes.
+\cgalFigureEnd
+
+While the user provides a distance
+the envelope is not the Minkowski sum with a sphere, because this would have cylindrical and spherical
```

Comment:
radius of the sphere = distance

Reply:
I would not say anything about convex edges but that the MS is the union of cylinders, sphere and prisms.

Reply:
I find it  instructive to see where the cylinders and spheres would come from.